src/components/plotlyChart.js

```diff
@@ -23,7 +23,13 @@ class PlotlyChart extends React.Component {
       .then((result) => {
 
         let dpJson = result.data
-        let dp = new ReclineView(dpJson)
+        let dp
```

Comment:
This seems repetitive of lineChart - suggests we should be factoring out somewhere common.